python/lsst/ts/vent/controller/controller.py

```diff
@@ -22,33 +22,49 @@
 import logging
 from enum import IntEnum
 
-import megaind
+try:
+    import megaind
```

Comment:
I missed the inclusion of this package dependency. I don't think this is part of our stack. Do you really need it? 

Usually, adding new package dependencies require approval by myself and/or Wouter. We should discuss this.

Reply:
I think it's necessary as a part of the hardware controlling the vents - the Sequent Microsystems Industrial Automation Hat.